api/libproject/devstart.rb

```diff
@@ -2373,7 +2373,7 @@ def with_optional_cloud_proxy_and_db(gcc, service_account = nil, key_file = nil)
     yield gcc
   else
     ENV.update(read_db_vars(gcc))
-    ServiceAccountContext.new(gcc.project).run do
+    ServiceAccountContext.new(gcc.project, service_account, key_file).run do
```

Comment:
`service_account` and `key_file` don't appear to be used set anywhere, so maybe these variables could be removed. While unlikely, I'm keeping it to match old functionality in case we do use them somehow.